### PR TITLE
update release instructions

### PR DESCRIPTION
**Pull request by patriknw**
Release train issue based on akka-grpc

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -154,5 +154,5 @@ that automatically merges the code, builds it, runs the tests and comments on th
 
 ## Source style
 
-akka-persistence-cassandra uses [Scalariform](https://github.com/scala-ide/scalariform) to enforce some of the code style rules.
+akka-persistence-cassandra uses [Scalafmt](http://scalameta.org/scalafmt/) to enforce some of the code style rules.
```
Comment (ennru) [inline CONTRIBUTING.md:157]:
```suggestion
akka-persistence-cassandra uses [Scalafmt](https://scalameta.org/scalafmt/) to enforce some of the code style rules.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,9 +26,9 @@ These guidelines mainly apply to Lightbend's "mature" projects - not necessarily
 
 Depending on which version (or sometimes module) you want to work on, you should target a specific branch as explained below:
 
-* `master` – development branch for Cassandra 3.x and Akka 2.5
-* `release-2.4` – development/release branch for Cassandra 3.x and Akka 2.4
-* `cassandra-2.x` – old development branch for Cassandra 2.x, but master can also be used with a Cassandra 2.x server (and we are currently only making releases from `master` and `release-2.4`) 
+* 1.0 -> `master` - current active development and stable 1.0.x patch releases
+* 0.80+ (currently 0.100) -> `release-0.x`  - removed use of Cassandra Materialized Views after they were marked as not to be used in production.
```
Comment (ennru) [inline CONTRIBUTING.md:30]:
```suggestion
* 0.80+ (see [Github releases](https://github.com/akka/akka-persistence-cassandra/releases) for latest) -> `release-0.x`  - removed use of Cassandra Materialized Views after they were marked as not to be used in production. 
```

File: README.md [hunk 3]
```diff
@@ -9,16 +9,16 @@ Replicated [Akka Persistence](https://doc.akka.io/docs/akka/current/scala/persis
...
 [![Build Status](https://travis-ci.org/akka/akka-persistence-cassandra.svg?branch=master)](https://travis-ci.org/akka/akka-persistence-cassandra)
 
-Implementation in the `master` branch is currently work-in-progress for the upcoming `1.0` release. [Snapshot documentation](https://doc.akka.io/docs/akka-persistence-cassandra/snapshot/) and [snapshot artifacts](https://oss.sonatype.org/content/repositories/snapshots/com/typesafe/akka/akka-persistence-cassandra_2.12/) are published for every successful `master` branch build.
+Implementation in the `master` branch is currently `1.0.x` release.
 
 
 ## Documentation
 
-Documentation for version 0.101 and above is available at https://doc.akka.io/docs/akka-persistence-cassandra/0.101/ with the latest docs on https://doc.akka.io/docs/akka-persistence-cassandra/current/.
+Current Documentation is available at https://doc.akka.io/docs/akka-persistence-cassandra/current/.
```
Comment (ennru) [inline README.md:17]:
```suggestion
The current version's documentation is available at https://doc.akka.io/docs/akka-persistence-cassandra/current/.
```

File: README.md [hunk 4]
```diff
@@ -29,11 +29,12 @@ It moved to the [Akka](https://github.com/akka/) organisation in 2016 and the fi
...
+There are three branches of development:
 
+* 1.0 -> `master` - current active development and stable 1.0.x patch releases
+* 0.80+ (currently 0.100) -> `release-0.x`  - removed use of Cassandra Materialized Views after they were marked as not to be used in production. 
 * 0.50+ (currently 0.62) -> `release-0.50`- first release under this organisation, previously under krasserm. No planned releases for this version.
-* 0.80+ (currently 0.100) -> `release-0.x`  - removed use of Cassandra Materialized Views after they were marked as not to be used in production. Current stable version.
-* 1.0 -> `master` - not yet released, planned upgrade to 4.0.x of the Cassandra Driver that includes breaking API changes to CassandraSession
 
-To release a 0.50+ or 0.80+ version checkout that branch and see the release instructions there. 
+## License
```
Comment (ennru) [inline README.md:38]: Add a note about support from Lightbend.

File: docs/release-train-issue-template.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+- [ ] For non-patch releases: Create a news item draft PR on [akka.github.com](https://github.com/akka/akka.github.com), using the milestone
+- [ ] Move all [unclosed issues](https://github.com/akka/akka-persistence-cassandra/issues?q=is%3Aopen+is%3Aissue+milestone%3A$VERSION$) for this milestone to the next milestone
+- [ ] Close the [milestone](https://github.com/akka/akka-persistence-cassandra/milestones?direction=asc&sort=due_date)
+
+### Cutting the release
+
+- [ ] Make sure any running [Travis builds](https://travis-ci.org/akka/akka-persistence-cassandra) for the commit you would like to release have completed.
+- [ ] Create the [release notes](https://github.com/akka/akka-persistence-cassandra/releases) with the next tag version `v$VERSION$`, title, release description and contributors (the latter generated by [`sbt-authors`](https://github.com/2m/authors) (eg. `sbt authors v1.0.1 HEAD`))
```
Comment (ennru) [inline docs/release-train-issue-template.md:31]:
```suggestion
- [ ] Create the [release notes](https://github.com/akka/akka-persistence-cassandra/releases) with the next tag version `v$VERSION$`, title, release description and contributors (the latter generated by [`authors`](https://github.com/2m/authors) (eg. `authors v1.0.1 HEAD`))
```

File: docs/release-train-issue-template.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+### Check availability
+
+- [ ] Check [reference](https://doc.akka.io/docs/akka-persistence-cassandra/$VERSION$/) documentation
+- [ ] Check the release on [Maven central](https://repo1.maven.org/maven2/com/typesafe/akka/akka-persistence-cassandra_2.12/$VERSION$/)
+
+### When everything is on maven central
+  - [ ] `ssh akkarepo@gustav.akka.io`
+    - [ ] update the `current` links on `repo.akka.io` by with the script:
+         ```
+         ./update-akka-persistence-cassandra-current-version.sh $VERSION$
```
Comment (ennru) [inline docs/release-train-issue-template.md:45]: Update the shell script to change the `1.0` link as well.

**Comment by patriknw** [comment]
> In many other repos we use the release drafter and autolabler, with that in place we don't really need to manage milestones manually. 

That feels scary. Problem with release drafter is that it marks PRs, not issues. I think it's very important to be able to see which version an issue was released in. The release notes should be a summary of the most important changes (so even if crafting it with release drafter it should be manually adjusted). Full list, including cross linking from the issues themselves, should be in the milestones.

File: docs/release-train-issue-template.md (hunk 6 above)
Comment (patriknw) [reply docs/release-train-issue-template.md:45]: 1.0 links to current or opposite

File: README.md (hunk 4 above)
Comment (patriknw) [reply README.md:38]: we have https://github.com/akka/akka-persistence-cassandra/issues/751 for that

**Pull request merged**